internal/web3ext/web3ext.go

```diff
@@ -838,14 +842,6 @@ web3._extend({
 			name: 'serverInfo',
 			getter: 'les_serverInfo'
 		}),
-		new web3._extend.Property({
-			name: 'gatewayFee',
-			getter: 'les_gatewayFee'
-		}),
```

Comment:
Was removing this an accident?

Reply:
im not sure when i added that but there is no gatewayFee method in the les api